### PR TITLE
comment for pr

### PR DESCRIPTION
**Pull request by Iqureshi223**
Story
====
__As an admin i want to edit our volunteers training requirements to confirm whether they are done or not__

---

1. admin view of edit profile should display training requirements
2. edit profile should be updated with yes or no value and this should be updated into dbpersons.

### CODE REVIEW

File: editProfile.php
Comment (noahstafford) [inline editProfile.php:1]: Line 18 lists /gwyneth, which we are not using anymore

File: editProfile.php [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <?php
     // Author: Lauren Knight
+    //comment this for ya.
     // Description: Profile edit page
```
Comment (noahstafford) [inline editProfile.php:4]: (it wont let me add comments to specific lines for some reason so i have to do it like this)
maybe remove code that has been commented out that we aren't planning on using, like gender and contact method

Comment (noahstafford) [inline editProfile.php:4]: Are errors being handled? Does it display a message to the user what specific fields are incorrect?

Comment (Iqureshi223) [reply editProfile.php:4]: i think we're detecting them and stopping them but i don't think we're leaving messages or comments about them. we should add those ya

File: editProfile.php
Comment (Iqureshi223) [reply editProfile.php:1]: got it

**Pull request closed without merging**